lumicks/pylake/channel.py

```diff
@@ -1093,3 +1100,43 @@ def channel_class(dset):
             raise IndexError("Direct access to this field is not supported.")
     else:
         return TimeSeries
+
+
+def match_sample_rates(slices: list[Slice], where="center"):
+    """Match sample rates across multiple slices by downsampling to the lowest rate.
+
+    Parameters
+    ----------
+    slices : list of Slice
+        List of slices to match sample rates for
+    downsampling_factor : int, optional
+        Additional downsampling factor to apply to all slices
```

Comment:
I think this was left over from the earlier implementation. Also, `where` is not documented.

```suggestion
    where : str
        Where to put the final time point.
        - "center" : The new time points are set to
          `(timestamps_subset[0] + timestamps_subset[-1]) / 2`, where `timestamps_subset` are
          the timestamps corresponding to the samples being downsampled over.
        - "left" : Time points are set to the starting timestamp of the downsampled data.
```